core/templates/dev/head/components/RatingComputationService.js

```diff
@@ -46,6 +46,14 @@ oppia.factory('RatingComputationService', [function() {
 
         return totalValue / totalNumber;
       }
+    },
+    getRelativeChangeInAverageRating: function(newRating, oldRating) {
+      if (!oldRating) {
+        return '+' + parseFloat(newRating).toFixed(2);
```

Comment:
Yes, you are right. I've always been using `toFixed` with `parseFloat` without realizing that it can be directly used with numbers as well!
Removed use of `parseFloat`.